src/main.rs

```diff
@@ -64,6 +66,15 @@ fn main() -> Result<()> {
         (xcb::CW_OVERRIDE_REDIRECT, 1),
     ];
 
+    // TODO: Move the ``window_cmd_map`` into the configuration
+    let mut window_command = &wm::WindowCommand::Focus;
```

Comment:
Hm I don't really like much that we have this `mut` command here as there's quite a bit of mental burden associated with this giant function and this variable could change at any time in this long block of code. Could you try to re-engineer that? Perhaps by making use of expressions in the block below where you default to `Focus` if no other key is found in the sequence.